### DISCUSSION TITLE
Support for Gencubes

### DISCUSSION THREAD

**Discussion by ItzFabb-repo** [BG-Software-LLC/WildTools#97] [Pending Feature Requests] [answered]
Hello, i wud like to request a support for gencubes plugin, atm when you mined a blocks inside gencubes it throws a bunch of errors :[

**Comment by OmerBenGera** [comment, marked as answer]
You can add "Geuncubes" to the other-plugins list in your config.
If you still get errors, open an issue and I'll look into that.
